ironic/drivers/modules/drac/management.py

```diff
@@ -257,6 +263,13 @@ def set_boot_device(node, device, persistent=False):
         raise exception.DracOperationError(error=exc)
 
 
+def get_lcc_out_of_recovery_mode(node):
+    client = drac_common.get_drac_client(node)
+    settings = {'Lifecycle Controller State': 'Enabled'}
+    client.set_lifecycle_settings(settings)
+    client.commit_pending_lifecycle_changes(reboot=False)
```

Comment:
Does this job run without rebooting the server?

Reply:
yes.